basket/news/backends/ctms.py

```diff
@@ -375,10 +377,10 @@ def _session(self):
             token_updater=self.save_token,
         )
         session.register_compliance_hook(
-            "access_token_response", CTMSSession.check_2xx_response
+            "access_token_response", CTMSSession.check_2xx_response,
```

Comment:
_precommit_

Reply:
sloppy. If precommit is needlessly changing code, then it should be disabled.